examples/frameworks/adk_demo/src/nat_adk_demo/register.py

```
@@ -0,0 +1,98 @@
+from . import weather_update_tool  # noqa: F401  # pylint: disable=unused-import
+
+logger = logging.getLogger(__name__)
+
+
+class ADKFunctionConfig(FunctionBaseConfig, name="adk"):
+    description: str
+    llm_name: str
+    tool_names: list[str] = Field(default_factory=list)
+
+
+@register_function(config_type=ADKAIQConfig, framework_wrappers=[LLMFrameworkEnum.ADK])
```

Ruff (0.12.2)

Undefined name `ADKAIQConfig`

(F821)